README.md

```diff
@@ -1,7 +1,10 @@
 go-buffer-pool
 ==================
 
-[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai)
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://protocol.ai)
```

Comment:
Theses should all be https (we probably need to normalize our repos).

Reply:
Done